src/components/TopBar/index.tsx

```diff
@@ -13,15 +13,17 @@ import { Popover, PopoverTrigger, PopoverContent, PopoverBody } from "@chakra-ui
 import { useStoreSelector } from "@/store/hooks";
 
 import ColorBoard from "./ColorBoard";
+import DownloadSvg from "@/assets/svg/TopBarSvg/download.svg";
 import BinSvg from "@/assets/svg/TopBarSvg/bin.svg";
 import DocSvg from "@/assets/svg/TopBarSvg/document.svg";
 import PaintBucketSvg from "@/assets/svg/TopBarSvg/paintBucket.svg";
 import UploadSvg from "@/assets/svg/TopBarSvg/upload.svg";
 import { useDispatch } from "react-redux";
 import { changeSelectedColor } from "@/store/reducer/courtColorSlice";
+import { downloadToPDF } from "../../utils/printPDF";
```

Comment:
Maybe try dynamic(lazy) load this util. If it can be lazy-loaded then we don't need to bundle this lib to our main js file after build.

Reply:
Cos it has 2 libraries in it. Lazy loading in any form would be better than bundle them all into our main exported file.

Reply:
Got it. I tried dynamic loading to import the function, but it was not allow. Cos dynamic import is used to import component. I will try another way.

Reply:
Done. Followed document from this link https://javascript.plainenglish.io/a-lazy-load-util-function-you-will-get-addicted-to-e682ab410ac8. using react/webpack approach.